test/user.spec.js

```diff
@@ -0,0 +1,114 @@
+// import chai from 'chai';
```

Comment:
Emeka, why do you have so much commented out codes?

Reply:
I was trying to solve a problem from a merge conflict. I will take care of now.